frontend/smart-kickers-game/src/components/Game/Header/StatsHeader.css

```diff
@@ -0,0 +1,3 @@
+.nav-buttons {
+  text-align: center;
+}
```

Comment:
`text-align` property should be used to align the content(text) of the element rather than multiple elements. I would suggest to use a `flex` here, it would be more elegant.

Reply:
Changed.